cli/src/cli.rs

```diff
@@ -42,7 +42,9 @@ use solana_faucet::faucet_mock::request_airdrop_transaction;
 use solana_rbpf::vm::{Config, Executable};
 use solana_remote_wallet::remote_wallet::RemoteWalletManager;
 use solana_sdk::{
+    account::Account,
```

Comment:
Cli changes could be a separate PR too

Reply:
Yeah, nice to see the instruction definitions together with how a user would interact with them